token/cli/src/config.rs

```diff
@@ -22,6 +22,8 @@ pub(crate) enum KeypairOrPath {
 }
 
 pub(crate) struct Config<'a> {
+    pub(crate) default_signer: Arc<dyn Signer>,
+    pub(crate) default_address: Pubkey,
```

Comment:
Since `default_address` is just `default_signer.pubkey()`, how about we get rid of it and rederive the pubkey whenever needed?